contextvh/src/main/java/contextvh/translators/J2ActionLog.java

```diff
@@ -36,11 +35,12 @@ public J2ActionLog() {
     public Parameter[] translate(final ActionLog actionLog) throws TranslationException {
         ParameterList parList = new ParameterList();
         ItemMap<Indicator> map = EventManager.<Indicator>getItemMap(MapLink.INDICATORS);
```

Comment:
Are you sure this works? Since 1.0.11 we had to add a slotConnectionID parameter to getItemMap calls.